OpenDreamRuntime/DreamValue.cs

```diff
@@ -138,9 +137,17 @@ public enum DreamValueType {
             }
         }
 
-        [Obsolete("Deprecated. Use TryGetValueAsFloat() instead.")]
+        public int MustGetValueAsInteger() {
+            if (Type != DreamValueType.Float) {
+                throw new Exception("Value " + this + " was not the expected type of integer");
```

Comment:
You can just get rid of these typecheck-and-throws in these Must methods. C# will emit its own runtime whenever it tries to unbox to the type and fails with a `InvalidCastException`.